MANIFEST.in

```diff
@@ -5,6 +5,7 @@ include .coveragerc
 
 recursive-include tests *.py *.whl deprecated-pypirc
 recursive-include docs *.bat *.empty *.py *.rst Makefile *.txt
+recursive-include .github *md
```

Comment:
Is this necessary to include?

Reply:
I've made requested changes for everything below.

As for including this line in the MANIFEST.in file: I did b/c Travis complained about my commit missing `.github` and `.github/ISSUE_TEMPLATE.md`.

After researching this further, I've now `python setup.py sdist` and removed line 8 from MANIFEST.in before my newest commit. OK to resubmit PR?

Reply:
Please just update this PR's branch. Please don't open an new PR